api/test/integration/test_security_GET_endpoints.tavern.yaml

```diff
@@ -234,6 +234,80 @@ stages:
         error: !anyint
         data:
           total_affected_items: 0
+
+  # GET /security/roles
+  - name: Try to show the roles of the system with a non-existent search
+    request:
+      <<: *all_roles_request
+      params:
+        search: notexist
+    response:
+      status_code: 200
+      json:
+        error: !anyint
```

Comment:
On new tests we must use the correct `error` code.

Reply:
Done